librad/src/meta/project.rs

```diff
@@ -79,6 +79,7 @@ impl Project {
 
     fn dedup_maintainers(&mut self) {
         let mut xs: Vec<PeerId> = self.maintainers.iter().cloned().collect();
+        xs.sort();
```

Comment:
It seems _sort_ of silly to `sort` every time to de-duplicate. Should we just keep the list in sorted order and use binary_search with insertions to maintain order and avoid insertions if they already exist?

Reply:
I think we want #16 at some point, but since this list is supposed to be small, the sort is negligible